scripts/add_a_method.sh

```diff
@@ -8,7 +8,7 @@ exit 1
 scripts/download_resources.sh
 
 # create a new component
-method_id="my_method"
+method_id="scape"
```

Comment:
Please undo these changes -- they should not be modified for everybody else to see :relaxed: 

```suggestion
method_id="my_method"
```